cli/Option.php

```diff
@@ -0,0 +1,100 @@
+<?php
+declare(strict_types=1);
+
+class Option {
+	public const VALUE_NONE = 'none';
+	public const VALUE_REQUIRED = 'required';
+	public const VALUE_OPTIONAL = 'optional';
+
+	private string $valueTaken = self::VALUE_REQUIRED;
+	/** @var array{type:string,isArray:bool} $types */
+	private array $types = ['type' => 'string', 'isArray' => false];
+	private string $optionalValueDefault = '';
+	private ?string $deprecatedAlias = null;
+
+	public function __construct(private readonly string $longAlias, private readonly ?string $shortAlias = null) {
```

Comment:
We are still supporting PHP 7.4+ a little longer (https://github.com/FreshRSS/FreshRSS/discussions/5474). This line does not seem compatible. Could you please fix?

Tip on how to test:

```sh
docker build --pull -f Docker/Dockerfile-Oldest --tag freshrss/freshrss:oldest .

docker run --rm -e FRESHRSS_ENV=development -v $PWD:/var/www/FreshRSS freshrss/freshrss:oldest sh -c 'bin/composer update && bin/composer test'

docker build --pull -f Docker/Dockerfile-Newest --tag freshrss/freshrss:newest .

docker run --rm -e FRESHRSS_ENV=development -v $PWD:/var/www/FreshRSS freshrss/freshrss:newest sh -c 'bin/composer update && bin/composer test'
```

Reply:
I've dealt with the issue with promoted properties along with a few other pieces related to 7.4 under [af0bbcb](https://github.com/FreshRSS/FreshRSS/pull/6099/commits/af0bbcbd61399a617198ca9c2ff7fdf44b487225).